scripts/generate-changelog.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# Generate a changelog for all PRs after the Lerna migration (git tag v7.7.0)
+# See docs at https://github.com/lerna/lerna-changelog
+lerna-changelog --from v7.7.0 > CHANGELOG.md
+
+# Add the pre-Lerna changelog to the bottom of the generated one.
+cat ./scripts/templates/CHANGELOG.legacy.md >> ./CHANGELOG.md
```

Comment:
I put the current changelog into a file called `CHANGELOG.legacy.md` and it gets added to the bottom of the fresh changelog that will be generated by `lerna-changelog` going forward.